src/resources/stubs/migrations/2014_02_04_000000_create_country_state_city_table.stub

```diff
@@ -17,6 +17,8 @@ class CreateCountryStateCityTable extends Migration
         Schema::create('countries', function (Blueprint $table) {
             $table->bigIncrements('id');
             $table->string('name',255);
+            $table->string('currency_code');
+            $table->string('calling_code');
```

Comment:
add string length $table->string('calling_code',255);

Reply:
some countries have multiple calling_code like US 
How do you handle this?

Reply:
For countries with multiple calling_code, the field includes the calling codes separated with commas

Reply:
I think comma separation is not a good idea, especially in laravel you can use array cast instead of comma-separated